fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -1990,6 +1992,14 @@ public static OnlineIndexer forRecordStoreAndIndex(@Nonnull FDBRecordStore recor
         return newBuilder().setRecordStore(recordStore).setIndex(index).build();
     }
 
+    protected List<Object> runAsyncDetails(String transactionName, Object ...keysAndValues) {
```

Comment:
(having `common.indexLogMessageKeyValues(String transactionName)` would also save the need for this double implementation)

Reply:
Yep, agreed. Done.